pkg/resolution/resolver/bundle/bundle.go

```diff
@@ -117,19 +123,28 @@ func GetEntry(ctx context.Context, keychain authn.Keychain, opts RequestOptions)
 					ResolverAnnotationName:       lName,
 					ResolverAnnotationAPIVersion: l.Annotations[BundleAnnotationAPIVersion],
 				},
+				source: &v1beta1.ConfigSource{
+					URI: uri,
+					Digest: map[string]string{
+						h.Algorithm: h.Hex,
+					},
+					EntryPoint: opts.EntryName,
+				},
 			}, nil
 		}
 	}
 	return nil, fmt.Errorf("could not find object in image with kind: %s and name: %s", opts.Kind, opts.EntryName)
 }
 
 // retrieveImage will fetch the image's contents and manifest.
-func retrieveImage(ctx context.Context, keychain authn.Keychain, ref string) (v1.Image, error) {
+func retrieveImage(ctx context.Context, keychain authn.Keychain, ref string) (string, v1.Image, error) {
```

Comment:
Thanks Chuang!
Can we also update the comment above for the return values?

Reply:
Good catch! Thanks @JeromeJu . Updated!